setup.py

```diff
@@ -15,8 +15,8 @@
 
 setup(
     name="foca",
-    version="0.0.1",
-    description="Archetype for flask-connexion microservices",
+    version="0.1.0",
```

Comment:
why not 0.0.1?

Reply:
I think it's more standard, because of the semantics associated with [semver](https://semver.org). Doesn't really make sense to start a project with a _patch_. On the other hand, a version of 1.x implies that a program is production-grade. So that's why the first version is typically 0.1.0. But it's not important, just convention, and others may disagree

Reply:
got it. :)